src/components/FormLabel/FormLabel.scss

```diff
@@ -5,5 +5,9 @@
   color: $color-font-base;
   display: block;
   font-weight: 700;
```

Comment:
In cases where we will eventual have tokens for these rules, should we create css vars in the meantime to make these easier to locate when it's time to replace?

We did this in logistics-app with the colors, so we could easily snap the app to the new style guide.

Reply:
@JCarmine Yes, I like this idea.